libs/payments/paypal/src/lib/paypalCustomer/paypalCustomer.manager.in.spec.ts

```diff
@@ -49,8 +49,7 @@ describe('PaypalCustomerManager', () => {
         billingAgreementId: 'NOT VALID LONG STRING TO CAUSE ERROR',
       });
 
-      expect.assertions(0);
-      expect(
+      await expect(
```

Comment:
I thought we needed `expect.assertions`? (https://jestjs.io/docs/tutorial-async#error-handling)

cc: @StaberindeZA

Reply:
It's not needed, no.

`expect.assertions` checks to make sure that a number of assertions have been made within the test. Using `expect.assertions(1)` is needed when using the `try { await asyncMethod() } catch(e) { expect(e).instanceOf(xyz)` syntax because if the method _doesn't_ throw, the test will still pass since the catch block is never called. Using the `await expect(asyncMethod()).rejects.toThrow(xyz)` automatically does this.

An addendum is that - at the bottom of the docs, they do offer that you _can_ use `expect.assertions(1)` _in addition to_ `await expect(asyncMethod()).rejects.toThrow(xyz)`. They suggest this as a catch for developers to help not forget to await the expect. Problem is, I don't think this really helps - 1. it's unnecessary, and 2. if the dev forgot the await call, they would almost certainly have forgotten the expect.assertions call, and 3. you have to count the number of expect calls and put that in the expect.assertions call.